test/e2e/signup.spec.js

```diff
@@ -8,6 +8,10 @@ describe('Nazgul front-end UI', () => {
     browser.waitForAngularEnabled(false);
   });
 
+  afterAll(() => {
```

Comment:
Does your tsk have anything to do with signup? Why are you editing the signup specs

Reply:
No, it doesn't have anything to do with signup. I was trying to debug. I have removed it.
I touched `landing.spec.js` because there was same id for two different divs and that caused the test to pass and fail sometimes. It was not consistent. So I told Gabriel and fixed it.